index.html

```diff
@@ -58,6 +60,171 @@ <h1 class="header-title">
+
+        <div class="work-item">
+            <img src="./assets/img/work/SnapshootPortfolio3.png" alt="tonic" width="295" height="220" class="work-item-image">
+            <div class="work-content">
+                <h3>Tonic</h3>
+                <div class="work-header-items">
+                    <p class="work-header-item-one work-header-item-margin">CANOPY</p>
+                    <span class="work-header-item-round work-header-item-margin"></span>
+                    <p class="work-header-item-two work-header-item-margin">Back End Dev</p>
+                    <span class="work-header-item-round work-header-item-margin"></span>
+                    <p class="work-header-item-two work-header-item-margin">2015</p>
+                </div>
+                <p class="work-pharagraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="work-tags">
+                    <a class="work-tag" href="#">html</a>
+                    <a class="work-tag" href="#">css</a>
+                    <a class="work-tag" href="#">javascript</a>
+                </div>
+                <button type="button" class="bg-white-button">See Project</button>
+            </div>
+        </div>
+
+        <div class="work-item">
+            <img src="./assets/img/work/SnapshootPortfolio4.png" alt="Multi-Post Stories" width="295" height="220" class="work-item-image">
+            <div class="work-content">
+                <h3>Multi-Post Stories</h3>
+                <div class="work-header-items">
+                    <p class="work-header-item-one work-header-item-margin">CANOPY</p>
+                    <span class="work-header-item-round work-header-item-margin"></span>
+                    <p class="work-header-item-two work-header-item-margin">Back End Dev</p>
+                    <span class="work-header-item-round work-header-item-margin"></span>
+                    <p class="work-header-item-two work-header-item-margin">2015</p>
+                </div>
+                <p class="work-pharagraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="work-tags">
+                    <a class="work-tag" href="#">html</a>
+                    <a class="work-tag" href="#">css</a>
+                    <a class="work-tag" href="#">javascript</a>
+                </div>
+                <button type="button" class="bg-white-button">See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <section class="about">
+        <h2 class="about-title">About <br> Myself</h2>
+        <p class="about-p">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+        <div class="social about-social">
+            <a class="social-title" href="#">LET’S CONNECT</a>
```

Comment:
- [ ] Since `hx` tags are for titles, texts like `LET'S CONNECT` should use a `hx` tag

Reply:
:+1: This issue has been addressed successfully :heavy_check_mark: